src/components/Sidebar.jsx

```diff
@@ -49,6 +42,21 @@ function Sidebar({ user, currentDBId, setCurrentDBId, setDocumentsIds }) {
     },
   );
 
+  async function deleteDatabase(databaseId) {
+    await fetchData("DELETE", `/users/${user.userId}/databases/${databaseId}`);
+  }
+
+  const { mutate: fetchDeleteDB } = useMutation(deleteDatabase, {
+    onSuccess: () => {
+      setCurrentDBId(databases[0]._id);
+      queryClient.refetchQueries(["dbDocumentList"]);
+      queryClient.refetchQueries(["userDbList"]);
```

Comment:
사용해주신 `refetchQueries`에 대해서 조사하다가, 만약 refetch 중에 에러가 발생하면 [에러가 throw 되지 않는다](https://tanstack.com/query/v4/docs/react/reference/QueryClient#queryclientrefetchqueries:~:text=By%20default%2C%20it%20will%20not%20throw%20an%20error%20if%20any%20of%20those%20queries%20refetches%20fail%2C%20but%20this%20can%20be%20configured%20by%20setting%20the%20throwOnError%20option%20to%20true)는 공식 문서 설명을 보았습니다.

> **Returns**
> This function returns a promise that will resolve when all of the queries are done being refetched. By default, it will not throw an error if any of those queries refetches fail, but this can be configured by setting the `throwOnError` option to `true`

혹시 refetch 실패 시에 대한 에러 핸들링이 이미 되어있는지 궁금하고, 만약 아니라면 추후 프론트엔드 에러 핸들링 때 같이 진행하면 좋을 것 같습니다 💯

Reply:
오 너무 좋은 포인트를 잡아주셨네요! 
현재 클라이언트 전반적으로 아직 Error 에 대한 처리가 되어 있지 않아 이 부분 유념하여 에러 핸들링때 사용하면 좋을듯 합니다!